examples/rest-collections/source/app.d

```diff
@@ -48,7 +48,8 @@ class LocalThreadAPI : ThreadAPI {
 
 	string[] get()
 	{
-		return m_data.threads.keys;
+		// NOTE: .keys is not @safe on older compiler versions
```

Comment:
Do you know from what version it is ?

Reply:
I tested it now on run.dlang.io and it works starting with 2.098.1.

Reply:
Rebased and added a `static if(__VERSION__)`.